main.py

```diff
@@ -43,7 +43,7 @@
 
 # Import config data
 import story_time.cc_creation as creation_messages
-from config import welcome_channel, game_master, dead_participant, frozen_participant, administrator
+from config import welcome_channel, game_master, dead_participant, frozen_participant, administrator, peasant
```

Comment:
what _is_ `peasant`? I see it all over the code, but can't find a reference

Reply:
Peasants are bots.